example/src/tests/groupTests.ts

```diff
@@ -932,3 +936,300 @@ test('can stream all group messages', async () => {
 
   return true
 })
+
+test('can streamAll from multiple clients', async () => {
+  const bo = await Client.createRandom({ env: 'local', enableAlphaMls: true })
+  await delayToPropogate()
+  const alix = await Client.createRandom({ env: 'local', enableAlphaMls: true })
+  await delayToPropogate()
+  const caro = await Client.createRandom({ env: 'local', enableAlphaMls: true })
+  await delayToPropogate()
```

Comment:
Added a helper method for client creation to make the tests less lines of code. It's been working fine in my tests without delayToPropogate in between, but feel free to modify test-utils/createClients.ts if you see error on client creation ( so far I have not).

```suggestion
const [alix, bo, caro] = await createClients(3)
```

Reply:
We can always clean up all the tests at once with the helper method in a separate PR if you prefer, but figured I'd let you know since it makes writing tests a little easier.

Reply:
Yep, saw your PR was waiting for it to merge I'll update to that

Reply:
Thank you for the example by the way, made it very easy to update without needing to track down additional code